components/inference-service/requirements.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+As data scientists & applications developers build AI models they need a way to deploy these applications in production. MONAI Inference Service will be used to deploy a MONAI application. This proposal documents the requirements for the MONAI Inference Service (MIS).
+
+## Goal
+The goal for this proposal is to enlist, prioritize and provide clarity on the requirements for MIS. Developers working on different software modules in MIS SHALL use this specification as a guideline when designing and implementing software for the Service.
+
+## Success Criteria
+MIS SHALL enable clients to communicate with it via a REST based RPC API.
+
+Clients SHALL be able to register the configuration of a [MONAI Application Packages(MAP)](https://github.com/Project-MONAI/monai-deploy/blob/main/guidelines/monai-application-package.md#manifests) with MIS. 
+
+Clients SHALL be able to submit a inference request to the server for running a job corresponding to a MAP along with an input dataset. 
+
+MIS SHALL service inference requests one at a time.
+
+MIS SHALL return inference results as part of the response to the originating inference request.
+
+## Requirements
+
+### Support for Specific MONAI Workloads
+MIS SHALL support [MONAI workloads](https://github.com/Project-MONAI/monai-deploy/blob/main/guidelines/monai-workloads.md#clinical-use-cases) which can be completed within the timeframe of a single HTTP request/response.
+
+### Deployable on MONAI Operating Environments
+MIS SHALL run on Staging Server and Production Server environments as defined in [MONAI Operating Environments](https://github.com/Project-MONAI/monai-deploy/blob/main/guidelines/MONAI-Operating-Environments.md).
+
+### API Interface
+MIS SHALL provide a REST based RPC API which utilizes the functionality of the HTTP transport.
+
+### Consistent and Robust Logging
+MIS SHALL provide consistent and robust logging about it's operations.
+
+### Register single MAP configuration before MIS startup
+MIS SHALL allow Clients to register the configuration of one MAP before MIS is started.
+
+### Fulfill an inference request with uploaded dataset
+MIS SHALL fulfill an inference request with uploaded input dataset.
+
+### Fulfill one inference request at a time
+MIS SHALL run only fulfill one inference request at a time.
```

Comment:
again - limitation, not requirement.

if we could fulfill multiple in parallel we would (and we maybe able to in v0.2 or later)